### DISCUSSION TITLE
How to change single `TextEdit` height

### DISCUSSION THREAD

**Discussion by willser** [emilk/egui#1541] [Q&A] [answered]
Hello guys.I got a question when I try to beautify my project.That is how to change single textEdit height.
I try following code:
```rust
ui.allocate_space()
```
But it doesn't work.

**Comment by emilk** [comment, marked as answer]
You can change `TextEdit::desired_rows` or make the font it uses larger

**Reply by willser** [reply]
Thank you,it's helpful.
